atak-civ/plugin-examples/plugintemplate/app/src/main/res/layout/create_timer_layout.xml

```diff
@@ -160,7 +193,8 @@
                 android:id="@+id/addNotificationButton"
```

Comment:
I feel like these buttons look good, but they could be hard to press on an actual phone. I wonder if we could make the hitbox a little bit wider or something? Up to you, but just something to think aobut